code/modules/mob/living/simple_animal/friendly/corgi.dm

```diff
@@ -570,6 +574,34 @@
 	response_harm   = "kicks"
 	spin_emotes = list("dances around.","chases his tail.")
 	is_pet = TRUE
+	var/creatine_had = 0
+	
+/mob/living/simple_animal/corgi/Ian/Life()
```

Comment:
why is this specific to Ian instead of all corgis?

Reply:
Ever since the other corgi changes I did were merged to normal corgis Ian has felt less special. This helps him be more special.
Also, if each corgi is producing 0.2 MW, that's like a lot of fucking power for only a chem dispenser and grinder with food in it.